### ISSUE TITLE
Able to talk in group Chat channels in spite of leaving the group

### ISSUE THREAD

**Issue by lycurgus** [urbit/urbit#3077]
**Describe the bug**
After leaving the Urbit Community group, the group and its chat channels are gone from Landscape but still appear via `;chats` in the CLI, and I can send/receive messages from them.

**To Reproduce**
1. Leave a group via Landscape
2. Go to Chat and note that the channels are correctly absent
3. Go to the CLI and note that the channels are still reported by `;chats`
4. Send a message to a group channel and note that it succeeds

**Expected behavior**
I shouldn't be able to send messages to channels that are in a group I'm no longer a member of.

**Desktop (please complete the following information):**
 - OS: Arch Linux
 - Browser: Firefox
 - Base hash of your urbit ship: 0vs.9288r.bdja5.ic2br.jigbg.0kduq.stbiu.npcsj.mtsk0.dhf80.l4gq1

**Additional context**
I was originally attempting to leave the Community group as part of a leave+rejoin, because I was able to join some chats (General, Design, Community Onboarding) but not others (Watercooler, Meetups & Events). It's possible this issue is just a symptom of whatever problem led to not being able to join those chats in the first place..?

**Comment by lycurgus** [comment]
Just as a note, I was seeing basically the same behaviour as #2857 in terms the chats that I was unable to join - a "past messages are being restored" and then "select, create or join a chat". Maybe my group state was already broken and trying to leave just broke it more?

**Comment by lycurgus** [comment]
By way of an update - ~simfur-ritwed re-added me to the group and it seems all is well now; I can access the channels that I wasn't previously able to. Apparently it may have been an issue with being in the old (dopzod) incarnation of the group, and those channels being ones that I joined via the old group.

I'm still not sure if it's expected behaviour that I was able to chat via CLI but not via Landscape - I'm guessing that Landscape goes "group-first" and just wasn't showing them, which feels like a bug but I guess a less serious one than access to a group chat by a non-member.
